js/renovation/ui/editors/check_box.tsx

```diff
@@ -192,6 +197,19 @@ export class CheckBox extends JSXComponent(CheckBoxProps) {
     return undefined;
   }
 
+  get iconStyles(): { [key: string]: string | number } {
+    const { iconWidth, iconHeight } = this.props;
+
+    const fontSizeMultiplayer = 0.73;
+    const computedIconWidth = typeof iconWidth === 'function' ? +iconWidth() : +iconWidth;
+    const computedIconHeight = typeof iconHeight === 'function' ? +iconHeight() : +iconHeight;
+    const width = normalizeStyleProp('width', typeof iconWidth === 'function' ? iconWidth() : iconWidth);
```

Comment:
Don't we really have an util to not call property function every time? Let's try to find and reuse it. If it does not exist, it's good idea to implement it

Reply:
created new util